Sources/ShopifyCheckoutSheetKit/CheckoutWebView.swift

```diff
@@ -203,10 +211,12 @@ extension CheckoutWebView: WKNavigationDelegate {
 		if isCheckout(url: response.url) && response.statusCode >= 400 {
 			CheckoutWebView.cache = nil
 			switch response.statusCode {
-			case 410:
-				viewDelegate?.checkoutViewDidFailWithError(error: .checkoutExpired(message: "Checkout has expired"))
+			case 403:
+				viewDelegate?.checkoutViewDidFailWithError(error: .checkoutUnavailable(message: "Forbidden."))
```

Comment:
This will now result in `.checkoutUnavailable` errors being thrown for checkouts disabled by killswitch